recipe/meta.yaml

```diff
@@ -8,13 +8,19 @@ source:
   fn: scikit-sparse-{{ version }}.tar.gz
   url: https://github.com/scikit-sparse/scikit-sparse/archive/v{{ version }}.tar.gz
   sha256: af850a0efc06576c6aa2d8a801316e9a2ae47609e48534b17352aac8b8deeba7
+  patches:
+    - 0001-Fix-import-in-setup.py.patch
```

Comment:
Could you please send this change upstream as well?

Reply:
This is already done: https://github.com/scikit-sparse/scikit-sparse/pull/84 :)